routes/authRoutes.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
+const User = require('../models/user')
+
+const app = express()
+app.use(express.json())
+
+// Sign up route
+app.post('/signup', async (req, res) => {
+    const { email, password } = req.body
```

Comment:
Does the mongo schema verify that (a) these fields are actually passed and (b) that they are valid? Just making sure we have that check somewhere.